exercises/collatz-conjecture/collatz_conjecture_test.f90

```diff
@@ -0,0 +1,35 @@
+
+! This test was created from E:\nosave\code\gitrepos\pclausen\problem-specifications\exercises\collatz-conjecture\canonical-data.json
+! version: None
+!
+
+program collatz_conjecture_test_main
+  use TesterMain
+  use collatz_conjecture
+  implicit none
+
+  ! Test 1: zero steps for one
+  ! UUID 1: 540a3d51-e7a6-47a5-92a3-4ad1838f0bfd
```

Comment:
The test UUIDs are mostly meant for track tooling. It's best to not expose them to the students and instead only leave the comments.